docker-jans-config-api/scripts/bootstrap.py

```
@@ -407,8 +407,11 @@ def import_ldif_files(self) -> None:
         # create missing scopes, saved as scopes.ldif (if enabled)
         logger.info("Missing scopes creation is enabled!")
         self.generate_scopes_ldif()
+        scope_file = "/app/templates/jans-config-api/scopes.ldif"
+        logger.info(f"Importing {scope_file}")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)